### DISCUSSION TITLE
How to integrate into html/webpage

### DISCUSSION THREAD

**Discussion by DayuanTan** [posit-dev/great-tables#333] [General]
After generating the table, do we save as png? could we integrate it with some html or webpages? Does it support plotly dash or other frameworks?

**Comment by jrycw** [comment]
@DayuanTan, you can use [GT.save](https://posit-dev.github.io/great-tables/reference/GT.save.html#great_tables.GT.save) to save the table in PNG format or [GT.as_raw_html](https://posit-dev.github.io/great-tables/reference/GT.as_raw_html.html#great_tables.GT.as_raw_html) to export the table in HTML format.

**Comment by jrycw** [comment]
I'll close the discussion since the issue appears to be resolved.
